apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/Components/TimeFrame.tsx

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import { TabSelector } from "@ledgerhq/native-ui";
-import { Dimensions, View } from "react-native";
+import { View } from "react-native";
 import { rangeDataTable } from "@ledgerhq/live-common/market/utils/rangeDataTable";
 import { useTranslation } from "react-i18next";
+import { KeysPriceChange } from "@ledgerhq/live-common/market/utils/types";
 
-export const TimeFrame = () => {
+type TimeFrameProps = {
+  range: KeysPriceChange;
+  setRange: (range: KeysPriceChange) => void;
+  width: number;
+};
+
+export const TimeFrame: React.FC<TimeFrameProps> = ({ range, setRange, width }) => {
   const { t } = useTranslation();
   const labels = Object.keys(rangeDataTable)
```

Comment:
you can type this to avoid doing `id => setRange(id as KeysPriceChange)` below

`const labels: YourType = Object...`